lib/resources/r4/immunization_create.yaml

```diff
@@ -0,0 +1,301 @@
+      }
+    }
+
+- name: lotNumber
+  required: 'No'
+  cardinality: 0..1
+  type: string
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.lotNumber
+  description: Lot number of the vaccine product.
+  action: create
+  example: |
+    {
+      "lotNumber": "123ab",
+    }
+
+- name: expirationDate
+  required: 'No'
+  cardinality: 0..1
+  type: date
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.expirationDate
+  description: Date vaccine batch expires.
+  action: create
+  example: |
+    {
+      "expirationDate": "2019-07-30"
+    }
+
+- name: site
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.site
+  description: Body site where vaccine was administered.
+  action: create
+  example: |
+    {
+      "site": {
+        "coding": [
+          {
+            "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/97",
+            "code": "576",
+            "display": "Left Arm",
+            "userSelected": false
+          }
+        ],
+        "text": "Left Arm"
+      }
+    }
+
+- name: route
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.route
+  description: The path by which the vaccine product is taken into the body.
+  action: create
+  example: |
+    {
+      "route": {
+        "coding": [
+          {
+            "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/4001",
+            "code": "318167",
+            "display": "IM",
+            "userSelected": false
+          }
+        ],
+        "text": "IM"
+      }
+    }
+
+- name: doseQuantity
+  required: 'No'
+  cardinality: 0..1
+  type: SimpleQuantity
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.doseQuantity
+  description: The quantity of vaccine product that was administered.
+  action: create
+  example: |
+    {
+      "doseQuantity": {
+        "value": 1.5,
+        "unit": "mL",
+        "system": "http://unitsofmeasure.org",
+        "code": "mL"
+      }
+    }
+
+- name: performer
+  required: 'No'
+  cardinality: 0..*
+  type: BackboneElement
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.performer
+  description: Indicates who performed the immunization event.
+  action: create
+  example: |
+    {
+      "performer": [
+        {
+          "function": {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/v2-0443",
+                "code": "AP",
+                "display": "Administering Provider"
+              }
+            ],
+            "text": "Administering Provider"
+          },
+          "actor": {
+            "reference": "Practitioner/7750958",
+            "display": "Houde, Darrell"
```

Comment:
change this to a different name here and below

Reply:
The practitioner Id belongs to Jake. So I added his name. https://github.com/cerner/fhir.cerner.com/pull/276/commits/6a039e09a31889bd9c0355262e750738c5a228a2#diff-6093e328274a82697fb90c9a404eda61R260

Reply:
My bad. Changed the names in all places. https://github.com/cerner/fhir.cerner.com/pull/276/commits/4596c4fb8faff62295dcbec728c1b72dc1a1f0b9